src/services/subject.js

```
@@ -27,6 +27,19 @@ const subjectService = {
       limit,
       totalPages: Math.ceil(total / limit)
     };
+  },
+
+  getSubjectById: async (subjectId) => {
+
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)